src/character.cpp

```diff
@@ -1296,13 +1293,12 @@ int Character::overmap_sight_range( float light_level ) const
     if( sight > 0 ) {
         sight = 6;
     }
-    // Mutations like Scout and Topographagnosia affect how far you can see.
-    sight += mutation_value( "overmap_sight" );
 
-    sight = enchantment_cache->modify_value( enchant_vals::mod::OVERMAP_SIGHT, sight );
+    sight = enchantment_cache->get_value_add( enchant_vals::mod::OVERMAP_SIGHT );
```

Comment:
```suggestion
    sight += enchantment_cache->get_value_add( enchant_vals::mod::OVERMAP_SIGHT );
```

I think that's how it should work.